docker/docker-compose-localnet.yaml

```diff
@@ -23,7 +23,7 @@ services:
       - 5001:5001
 
   lotus:
-    image: textile/lotus-devnet:ntwk-calibration-8.8.0
+    image: textile/lotus-devnet:ntwk-calibration-8.13.1.f0942
```

Comment:
Just noticing this version is different than the one in `docker/Makefile`, making sure that is intended.